src/sentry/templates/sentry/auth-link-identity.html

```diff
@@ -11,7 +11,13 @@
   {% csrf_token %}
 
   <div class="align-center">
+    {% if organization.avatar.all|length %}
```

Comment:
We would want to do this outside of template code, since this is relatively expensive, especially since this is being run twice.

Reply:
Can you link to the file where to add this? I tested this code in our sentry installation, but would be very interested in the correct solution - free to learn python here ;)!

Reply:
Can you elaborate why this is more expensive doing this in the template?

I mean the twig template is compiled and cached, right?
Unless you explicitly cache the result of the avatar somewhere (which should be done of course not in the template) it should remain the same.

Note that I don’t know the internals of this library 👍🏻

Reply:
I don't think the templates are compiled and cached. I'll see how we can improve this.